File: Friends.NET/Friends.Common/Data/Models/ApiFriend.cs
using System;
namespace Friends.Common.Data.Models
{
	public class ApiFriend
	{
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string NickName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string ProfilePictureUrl { get; set; }
        public bool IsFriend { get; set; }
        public int MajorsWon { get; set; }

    }
}

File: Friends.Kotlin/app/src/main/java/com/friends/data/models/ApiFriend.kt
package com.friends.data.models

import com.squareup.moshi.JsonClass
import java.time.Instant

@JsonClass(generateAdapter = true)
data class ApiFriend(
    val firstName: String,
    val lastName: String,
    val dateOfBirth: Instant?,
    val nickName: String
)
